onward/app/controllers/RenderTemplateController.scala

```diff
@@ -0,0 +1,36 @@
+package controllers
+
+import common.{Edition, ExecutionContexts, JsonComponent, Logging}
+import contentapi.ContentApiClient
+import implicits.Requests
+import model.{ApplicationContext, Cached, NoCache}
+import play.api.mvc._
+import play.twirl.api.HtmlFormat
+
+abstract class RenderTemplateController(contentApiClient: ContentApiClient)(implicit context: ApplicationContext) extends Controller with Paging with Logging with ExecutionContexts with Requests {
```

Comment:
Not sure the name of this class tells me much about what it is.  How about `OnwardContentCardController`?